server/src/main/scala/ch.epfl.scala.index.server/DataRepository.scala

```diff
@@ -478,6 +500,20 @@ class DataRepository(github: Github, paths: DataPaths)(
       .map(_.totalHits)
   }
 
+  def contributingProjects() = {
+    esClient
+      .execute {
+        search(indexName / projectsCollection)
+          .query(
+            functionScoreQuery(contributingQuery)
+              .scorers(randomScore(scala.util.Random.nextInt(10000)))
```

Comment:
would be good to double check if this means `a + b + c + random` where a,b, c

```
existsQuery("github.beginnerIssues")
existsQuery("github.contributingGuide")
existsQuery("github.chatroom")
```

are a, b, c values between 0 and 1? This would make 1 + 1 + 1 + large random value.